From ac11af06313e50086a18982a1a2189b3cde13d2d Mon Sep 17 00:00:00 2001
Message-ID: <ac11af06313e50086a18982a1a2189b3cde13d2d.1734811274.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1734811274.git.sharlatanus@gmail.com>
References: <cover.1734811274.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 16 Dec 2024 21:54:53 +0000
Subject: [PATCH 17/38] gnu: python-asdf: Update to 4.0.0.

* gnu/packages/astronomy.scm (python-asdf): Update to 4.0.0.
[propagated-inputs]: Add python-packaging.
[native-inputs]: Remove python-packaging.

Change-Id: I8a26d7e47c401a14c612450e4d3dd68d17f64e7e
---
 gnu/packages/astronomy.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index a6f363772f..8254c035cd 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1601,13 +1601,13 @@ (define-public python-aplpy
 (define-public python-asdf
   (package
     (name "python-asdf")
-    (version "3.3.0")
+    (version "4.0.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "asdf" version))
        (sha256
-        (base32 "0pwy8p3idz40a1z07d0lvvd0mwwya8g52wrp72frgpagjkvj3ska"))))
+        (base32 "1nzrgsyidcynhgiqxy7pwny07prza1g6b7hs17ybg6fh9zas6kc6"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -1623,7 +1623,6 @@ (define-public python-asdf
                 (("version_file = \"asdf/_version.py\"") "")))))))
     (native-inputs
      (list python-fsspec
-           python-packaging
            python-psutil
            python-pytest
            python-pytest-doctestplus
@@ -1639,6 +1638,7 @@ (define-public python-asdf
            python-jmespath
            python-lz4
            python-numpy
+           python-packaging
            python-pyyaml
            python-semantic-version))
     (home-page "https://github.com/asdf-format/asdf")
-- 
2.46.0

